doc/user.rst

```diff
@@ -230,3 +234,136 @@ The following JSON snippet shows an example of a network configuration with dyna
   }
 
 In the following example, SEE will generate a subnetwork within the network 192.168.0.0/16. The subnetwork will have the address 192.168.X.0/24 where X is a random number in the range 0-255. The DHCP server will assign addresses to the sandbox in the range 192.168.X.[0-255].
+
+Image Providers
++++++++++++++++
+
+SEE `image_providers` defines a simple common interface to interact with backend disk image providers. Their configuration may vary depending on the chosen provider and backend, however the `image` configuration section must hold a dictionary with the following structure, common to all providers:
```

Comment:
This is semantics, depending on whether `image_providers` is the interface or the implementations. The plural form may be less controversial though.